spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -102,6 +102,22 @@ def rescale_actions(actions, action_thresh=0.05, silence_only=False):
     return actions
 
 
+def rescale_arm_ee_actions(actions, action_thresh=0.05, silence_only=False):
+    actions = np.clip(actions, -1, 1)
+    # Silence low actions
+    actions[np.abs(actions) < action_thresh] = 0.0
+    if silence_only:
```

Comment:
we can remove this and just use `rescale_actions()`, there is no difference in the two functions